### ISSUE TITLE
“Startle”animation not playing

### ISSUE THREAD

**Issue by hyjhkt** [SmartlyDressedGames/Unturned-3.x-Community#4410]
I have a long-range attacking animal
When my animation is named "Startle" and there is no "Startle-Anim_Variants 1" in the dat, the animal does not play the Startle action before starting the attack, from the "Walk" or "Run" action to the "Attack" action.
When my animation is named "Startle-0" and there is "Startle-Anim_Variants 1" in the dat, the animal does not play the Startle action before starting the attack.

**Comment by MoltonMontro** [comment]
Could you perhaps post a screenshot of your asset in Unity, and copy-paste the contents of your .dat file?

**Comment by hyjhkt** [comment]
@MoltonMontro @SDGNelson 
![0135OMJ1@WMARQL~N X A@S](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/82864808/a228a880-f0ef-4c25-89f6-8ef648fa04b2)
![G_X9(QK`DNS`US_N%1VD5H](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/82864808/085966d5-cf9c-411d-b52d-61a27f519ec7)
![5ALE}A2{T$2`M`NDTUJFF_L](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/assets/82864808/1d25b3ca-bbf7-4e21-9ca3-498e654b2a4f)

**Comment by SDGNelson** [comment]
It sounds like it probably starts attacking immediately. Is that correct? Attack currently takes priority over the startle animation, and since it has a long range it would be able to attack as soon as it detects the player.

**Comment by hyjhkt** [comment]
@SDGNelson Is there a way to solve it

**Comment by SDGNelson** [comment]
Not at the moment, sorry. Animal improvements are currently low priority (not in the top 5 voted for this year) so I'm not sure when I'll next be working on them.
